app/javascript/Pages/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { Link } from "@inertiajs/inertia-react";
+import { FaBars, FaTimes } from "react-icons/fa";
+import "./Navbar.css";
+import { Button } from "../Button";
+
+function Navbar(props) {
+  const token = document.querySelector("[name=csrf-token]").content;
+  const headers = { "X-CSRF-Token": token };
+
+  const [click, setClick] = useState(false);
+  const [button, setButton] = useState(true);
+
+  const handleClick = () => setClick(!click);
+  const closeMobileMenu = () => setClick(false);
+
+  const showButton = () => {
+    if (window.innerWidth <= 960) {
+      setButton(false);
+    } else {
+      setButton(true);
+    }
+  };
+
+  const show = props.current_user;
+
+  window.addEventListener("resize", showButton);
```

Comment:
**question (blocking)**: why did you decide to do this with javascript? a CSS media querie can do this with much better performance. The resize event can run up to 60 times per second when it triggers 😬

Reply:
I did that first but then I used the variable show as a condition, so I will delete this 😬